.github/ISSUE_TEMPLATE/add-orbit-chain-request.yml

```diff
@@ -268,3 +269,12 @@ body:
       label: Child WETH Gateway
     validations:
       required: true
+
+  - type: input
```

Comment:
Maybe this can be optional and they can leave it empty in case the chain is already live? we can mention this in the issue